### PR TITLE
Telemetry off env parameter

### PR DESCRIPTION
**Pull request by thomas-gerber**
# Description

Adds a parameter that disable telemetry completely before starting Faros CE.

## Type of change
- New feature (non-breaking change which adds functionality)

# Checklist
(Delete what does not apply)
- [x] Have you checked to there aren't other open Pull Requests for the same update/change?
- [x] Have you lint your code locally before submission?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

**Comment by thomas-gerber** [comment]
Before merging, I would like to understand why init failed for our user - it shouldn't have AFAICT: https://faroscommunity.slack.com/archives/C0335HMN2NQ/p1677263489918959

File: init/src/airbyte/init.ts [hunk 1]
```diff
@@ -307,6 +307,7 @@ async function main(): Promise<void> {
     .requiredOption('--airbyte-destination-hasura-url <string>')
     .requiredOption('--hasura-admin-secret <string>')
     .option('--force-setup')
+    .option('--telemetry-off')
```
Comment (tovbinm) [inline init/src/airbyte/init.ts:310]: Commander recommends `--no-telemetry` instead

Comment (thomas-gerber) [reply init/src/airbyte/init.ts:310]: noted

Comment (tovbinm) [reply init/src/airbyte/init.ts:310]: and then you can call you member in options class simply `telemetry`

**Review by tovbinm — APPROVED** [review]

**Comment by thomas-gerber** [comment]
Closing for now without merging because:
1. we haven't root-caused why the error isn't caught (it is probably not super mysterious for someone familiar with async calls)
2. we are re-doing the whole init anyway for Airbyte v40

I'll keep the branch around until that work is done just in case someone else faces this issue.

**Pull request closed without merging**